src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareOperation.cs

```diff
@@ -118,6 +118,15 @@ public void Execute(TaskExecutionMode mode)
                         this.Differences.Add(diffEntry);
                     }
                 }
+
+                // HashSet contains the set of errors that are stopping the schema compare
+                // Error Message support single message only, so breaking the functionality with single error each time
```

Comment:
could you append the error messages so not just the last error message will be returned?